scripts/deploy.sh

```diff
@@ -50,10 +50,20 @@ DEPLOY_TYPE="soft"
 
 check_environment(){
   CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
+  CURRENT_TAG=`git tag --points-at HEAD`
+
+  # If we are not on master, we can still deploy if we have a valid tag
+  # The CI jobs are based on a tag (not a branch)
+  HARD_DEPLOY_TESTING_TAG_REGEX="^(testing_)?v[0-9]+(\.[0-9]+){2}$"
 
   if [[ "$APP_ENV" == "testing" && "$CURRENT_BRANCH" != "master" ]];
```

Comment:
❓ est-ce qu'on veut quand même pouvoir faire les MES sur master ? ou on ne garde que les tags ?

Reply:
staging et prod ne sont pas touchés pa ce changement parce que `"$APP_ENV" == "testing"` est False => du coup on pas directement à `Deploying to $APP_ENV.`

Reply:
oké ! du coup le but est de poouvoir déployer des tags (et pas que sur master) en testing uniquement. 
Mais pourquoi la CI ne déploie pas master ? j'ai du mal à comprendre le besoin et qu'est ce qui a changé qui nécessite ce changement

Reply:
OK vu au tel : maintenant on déclenche le build hard avec la création d'un tag vs détection auto du besoin au merge sur master avant